cg/meta/archive/ddn_dataflow.py

```diff
@@ -162,7 +166,7 @@ def _set_auth_tokens(self) -> None:
             ).model_dump(),
         )
         if not response.ok:
-            raise DdnDataflowAuthenticationError(message=response.content)
+            raise DdnDataflowAuthenticationError(message=response.text)
```

Comment:
Out of scope, but we should not automatically assume it is an authentication issue for any response status other than 200

Reply:
Authentication error would be 401

Reply:
Good point! Will create a small PR for this in the future.